From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: pthier <pthier@chromium.org>
Date: Mon, 2 Jan 2023 12:02:33 +0100
Subject: Explicitly include ICUs utf16.h

... required for U16_NEXT in builds with ICU.

Change-Id: I2ccda0c3fa7fd0139745e6233c6ab4a5dec46b50
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/4128520
Commit-Queue: Patrick Thier <pthier@chromium.org>
Reviewed-by: Victor Gomes <victorgomes@chromium.org>
Cr-Commit-Position: refs/heads/main@{#85051}

diff --git a/src/regexp/regexp-parser.cc b/src/regexp/regexp-parser.cc
index f420b2eec42d413941751e0d2831689fa820e260..f3f36bf1673d5bce320bf23cd5b3ff338d0891c9 100644
--- a/src/regexp/regexp-parser.cc
+++ b/src/regexp/regexp-parser.cc
@@ -20,6 +20,7 @@
 #include "unicode/uniset.h"
 #include "unicode/unistr.h"
 #include "unicode/usetiter.h"
+#include "unicode/utf16.h"  // For U16_NEXT
 #endif  // V8_INTL_SUPPORT
 
 namespace v8 {
